src/main/resources/templates/macros/page/navigation.vm

```diff
@@ -7,9 +7,9 @@
         #if ($run_with_jenkins)
           <li role="presentation"><a href="../">Jenkins</a></li>
           #if ($build_previous_number)
-            <li role="presentation"><a href="../../$build_previous_number/cucumber-html-reports/$report_file">Previous results</a></li>
+            <li role="presentation"><a href="../../$build_previous_number/cucumber-html-reports$directory_qualifier/$report_file">Previous results</a></li>

```

Comment:
did you address my concern from last review about compatibility?

Reply:
Sorry, I'm not sure which one you are referring to. If this is about navigation to previous/latest, then this particular change is exactly for that.

Reply:
Previous link does not work after this change